app/config/variables.php

```diff
@@ -119,6 +119,14 @@
                 'required' => false,
                 'question' => '',
             ],
+            [
```

Comment:
@TorstenDittmann with #947, maybe we can remove this option to reduce complexity?